docs/FAQ.md

```diff
@@ -96,10 +96,17 @@ The command will need to be run each time the host is restarted.
 
 Remember: the security profile for the container will need to allow traffic to the exposed port as well.  You can read about how to configure security profiles in the [Advanced Network Policy](AdvancedNetworkPolicy.md) guide.
 
-### Can I run Calico in a public cloud environment? 
+### Can I run Calico in a public cloud environment?
 Yes.  If you are running in a public cloud that doesn't allow either L3 peering or L2 connectivity between Calico hosts then you can specify the `--ipip` flag your Calico IP pool:
 ```
 ./calicoctl pool add <CIDR> --ipip --nat-outgoing
 ```
 Calico will then route traffic between Calico hosts using IP in IP.
+
+In AWS, you can instead disable `Source/Dest. Check` as long as all your instances are in the same subnet of your VPC.  This will provide the best performance.  You can disable this with the CLI, or in the `Networking` part of the instances' right click menus.
```

Comment:
Nitpick: agreed, active is better.

As for how it will affect, I'm going to assert that I have: it "will provide the best performance".  If we're going to explain anything in more detail, it should be IPIP.  I don't want to create FUD where there's really no grounds for it by needlessly explaining things.